### DISCUSSION TITLE
[V5]404 insted of 401 auth route

### DISCUSSION THREAD

**Discussion by rodolphonetto** [adonisjs/core#1632] [Help] [answered]
Hi guys, I'm trying to reproduce the docs example on auth routes but when I dont pass a token I'm receiving a 404 

`E_ROUTE_NOT_FOUND: Cannot GET:/login`

shouldn't I receive the 401? "E_INVALID_API_TOKEN"?

```
Route.get("dashboard", async ({ auth }) => {
    await auth.authenticate();

    return {
        hello: "world",
    };
});
```

**Comment by thetutlage** [comment, marked as answer]
Coz the `authenticate` call redirects you to the `/login` page. Do you have `/login` route?

**Reply by rodolphonetto** [reply]
Problem solved :)
